src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -152,6 +153,31 @@ public bool IsVisible
         /// </remarks>
         public bool ShouldSwapAutomatically { get; }
 
+        /// <inheritdoc />
+        public unsafe VideoMode VideoMode

```

Comment:
Also, couldn't we just use the IMonitor property to obtain this value rather than duplicate the native transition code?

Reply:
I changed it to use the Monitor property.